chromium_src/chrome/browser/plugins/plugin_info_message_filter.cc

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
Is this class used anywhere ?, since `WidevineCdmMessageFilter` seems to be taking care of the job for widevine plugins.